ext/ffi/ffi.c

```diff
@@ -4452,6 +4461,39 @@ ZEND_METHOD(FFI, isNull) /* {{{ */
 }
 /* }}} */
 
+ZEND_METHOD(FFI, hasSym) /* {{{ */
+{
+	zend_string *symbol;
+	zend_long symbol_kind;
+	zend_ffi_symbol *sym = NULL;
+	bool kind_is_null = 1;
+
+	ZEND_FFI_VALIDATE_API_RESTRICTION();
+	ZEND_PARSE_PARAMETERS_START(1, 2)
+		Z_PARAM_STR(symbol);
```

Comment:
The rest of the code supports calling the method statically and non-statically. The parameter parsing needs to cater to that.

Reply:
The method is non-statically method,  do not call by statically

Reply:
If the method is supposed to always be called non-statically, why is there a need for the special handling in `zend_ffi_get_func()`? (AFAIK, this is only needed for methods which can be called statically *and* non-statically.)

Reply:
Because zend_ffi_get_func() is trampoline of call C function, so need exclude

Reply:
Oh, right, thanks!